custom_components/econet300/sensor.py

```diff
@@ -151,27 +148,41 @@ def create_mixer_sensor_entity_description(key: str) -> EconetSensorEntityDescri
     return entity_description
 
 
-def create_mixer_sensors(coordinator: EconetDataCoordinator, api: Econet300Api):
+# Dynamically generate SENSOR_MIXER_KEY
+SENSOR_MIXER_KEY = {
+    str(i): {f"mixerTemp{i}", f"mixerSetTemp{i}"}
+    for i in range(1, AVAILABLE_NUMBER_OF_MIXERS + 1)
+}
+
+
+def create_mixer_sensors(
+    coordinator: EconetDataCoordinator, api: Econet300Api
+) -> list[MixerSensor]:
     """Create individual sensor descriptions for mixer sensors."""
     entities: list[MixerSensor] = []
-    # TODO: Cleanup logic add mixer only which not null in endpoint
+
     for i in range(1, AVAILABLE_NUMBER_OF_MIXERS + 1):
```

Comment:
This for cycle should be removed, you should cycle based on your created variable 

SENSOR_MIXER_KEY